.github/workflows/deploy.yml

```
@@ -62,4 +62,4 @@ jobs:
           path: "./build"
       - name: Deploy to GitHub Pages
         id: deployment
-        uses: actions/deploy-pages@v2
+        uses: actions/deploy-pages@v3
```

The suggestion wasn't to use an older version of this action.
```suggestion
        uses: actions/deploy-pages@v4
```

Instead, bump the version of `actions/upload-pages-artifact` on line 60.